libraries/adaptive-expressions/src/builtinFunctions/dateFunc.ts

```diff
@@ -31,9 +32,14 @@ export class DateFunc extends ExpressionEvaluator {
     private static evaluator(): EvaluateExpressionDelegate {
         return FunctionUtils.applyWithError(
             (args: any[]): any =>
```

Comment:
Can this be `args: unknown[]` for better type safety?

Reply:
Yes. About the abuse of `any` type, we would have a separate issue/fix to track. Personally think that a unified modification method would be better

Reply:
Agreed, so far I've just been mentioning it during code review.